### DISCUSSION TITLE
One of "My Tracks" keeps disappearing from the map view

### DISCUSSION THREAD

**Discussion by frobijn** [osmandapp/OsmAnd#11818] [Q&A]
### Description
I've got two GPX-tracks (A and B) that are displayed all the time on the map. At seemingly random moments, when I start using OsmAnd after a day or so, track A has disappeared from the map view but track B is still displayed. The track will reappear if I go to "My Places" and select "show track": all other settings (line color etc) are still the same. OsmAnd only seems to forget that I turned on the "show" for the track.

### How to reproduce?
I can't figure out what triggers the bug:

- Size: track A is 18MB, track B is 4MB
- Update: both tracks are frequently updated on a PC, the updated version is copied via USB-connection to the phone. Sometimes it seems to be this action that hides track A.
- Application (deep) sleep mode: it never happens when I'm using the app multiple times within an hour, but can happen when I start using the app after a day or so.

This started wen the new synchronization features were rolled out some months ago. I'm willing to experiment to find a reliable way to reproduce this behavior, but I don't know where to start. 

### Your Environment
OsmAnd Version: 3.9.10
Android/iOS version: Android 11
Device model: Samsung S10

**Comment by vshcherb** [comment]
I guess it's related to the size 18 MB :-(  Does complete application restart help?
